components/brave_sync/brave_sync_prefs.cc

```diff
@@ -147,6 +150,18 @@ bool Prefs::IsSyncV1Enabled() const {
   return pref_service_->GetBoolean(kSyncEnabled);
 }
 
+void Prefs::ClearSyncV1Enabled() {
```

Comment:
Seems like this is not getting called anymore.

Reply:
that's right, I'll remove